.github/workflows/changesets.yml

```
@@ -39,7 +39,15 @@ jobs:
       - name: Check the changesets
         run: node -r esbuild-register tools/deployments/validate-changesets.ts
 
-      - name: Check the build
+      - name: "Create .env file"
```

NIT: the description of this step is not very helpful. Can we describe the intent instead?

```suggestion
      - name: "Configure turbo repo cache environment variables"
```